app/models/restaurant.rb

```diff
@@ -13,7 +13,10 @@ def self.for_markers
     all.to_a.map(&:serializable_hash).map do |restaurant|
       {name: restaurant['name'],
        latitude: restaurant['latitude'],
-       longitude: restaurant['longitude']}
+       longitude: restaurant['longitude'],
+       description: restaurant['description'],
+       restaurant_category: restaurant['restaurant_category'],
```

Comment:
Does that return anything useful? What are you using it for?

Reply:
Ooops, I forgot to remove that. We we're just testing something.